packages/core-web/src/index.js

```diff
@@ -83,7 +83,22 @@ function detectAndApplyFixedHeaderStyles() {
     });
     resizeObserver.observe(headerSelector[0]);
   };
+
+  let lastOffset = 0;
+  const toggleHeaderOnScroll = () => {
+    if (window.innerWidth > 767) { return; }
+    const currentOffset = window.pageYOffset;
+    if ((window.innerHeight + currentOffset) >= document.body.offsetHeight) { return; }
```

Comment:
Hmm I think we can add some comment here as to why this is needed. I wasn't quite sure what this was for until I tested it out. And maybe we can assign it to a variable as well, maybe something like `isEndOfPage`.

Reply:
Yup I agree with @wxwxwxwx9 . 

Other than that everything LGTM! Thanks for making the changes @jonahtanjz !

Reply:
Thanks for the suggestion @wxwxwxwx9 and @raysonkoh. Have added the comments :)